src/vorta/views/archive_tab.py

```diff
@@ -714,12 +715,24 @@ def extract_action(self):
         else:
             self._set_status(self.tr('Select an archive to restore first.'))
 
+    def get_extract_display_mode(self):
```

Comment:
The name isn't fully fitting. The method doesn't return the display mode but instantiates a TreeModel.